test/utilities.py

```diff
@@ -6,7 +6,7 @@
 from urllib.parse import parse_qs, urlparse
 
 LOCAL_WEBSERVER_PORT = 8000
-BASE_TEST_URL_DOMAIN = "localtest.me"
+BASE_TEST_URL_DOMAIN = "localhost"
```

Comment:
Please, also update the docstring of the `start_server()` function in this file, to reference `http://localhost:8000/test_pages/...` instead of `http://localtest.me:8000/test_pages/...`

Reply:
Done